<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>881</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 0);
background-color: rgb(144, 229, 255);</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>190</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Communication Settings</string>
    </property>
    <widget class="QPushButton" name="ConnectBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SelPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(118, 250, 246);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SelBR">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Logging</string>
    </property>
    <widget class="QPushButton" name="PlotLoggedData_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot Logged data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenFile_btn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Log_checkBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Enable Logging</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Log file location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Log_Time_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="PercentageOpen_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Valve Percentage Opening&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Inlet Pressure (Bar)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="InletPress_Lable">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Outlet Pressure (Bar)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="OutletPress_Lable">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;SetPoint (Bar)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SetPoint_Lable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="UP_Btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DN_Btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Left Click -1, Right click -10</string>
    </property>
    <property name="text">
     <string>DN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; text-decoration: underline; color:#0055ff;&quot;&gt;Air Regulator Controller&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EXITBtn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetSpBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_Use"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuHelp"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHow_to_Use">
   <property name="text">
    <string>How to Use</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
